server/monitor/credentials.go

```diff
@@ -0,0 +1,21 @@
+package monitor
+
+import (
+	"time"
+)
+
+func (m *Monitor) checkCredentials(force bool) {
+	if !force && (time.Now().Hour() != 0 || time.Now().Minute() != 0) {
+		return
+	}
+
+	credentials, err := m.client.GetAppCredentials(m.applicationID)
```

Comment:
Can we skip trying if `applicationID` is empty? (With a debug noting same.) It would also be ideal if we could distinguish a generic error from one that relates to not having the credentials to make this request. Is this possible?

Reply:
Done